CoreWebServer/src/au/gov/asd/tac/constellation/webserver/api/swagger-ui/index.html

```diff
@@ -34,21 +34,70 @@
     <div id="swagger-ui"></div>
 
     <script src="./swagger-ui-bundle.js"> </script>
-    <script src="./swagger-ui-standalone-preset.js"> </script>
+    <script src="./swagger-ui-standalone-preset.js"> </script>    
+    
     <script>
-    window.onload = function() {
+    
+    window.onload = function() {        
+        //experiment components wrapping       
+        
+        const MyApiKeyAuthPlugin = function() {
+        return {
+          statePlugins: {
+            auth: {
+              wrapActions: {
+                  authorize: (oriAction, system) => (payload) => {
+                      //get currently running server:port
+                  let currServer = system.oas3Selectors.selectedServer();                  
+                  let err = system.errActions;
+                  //clear any existing auth errors
+                  err.clear({ source: "auth" });
+                  if(payload && payload.ConstellationSecret && payload.ConstellationSecret.value) {
+                      //call any example endpoint to check authorization with entered secret
+                      fetch(`${currServer}/v2/service/reflect_json`, {
+                        method: 'POST',
+                        headers: {
+                          'Content-Type': 'application/json',                      
+                          'accept': '*/*',
+                          'X-CONSTELLATION-SECRET':payload.ConstellationSecret.value
+                        }
+                      }).then(response => {
+                          // Check if the response was successful (e.g., status code 200-299)
+                          if (!response.ok) {
+                              err.newAuthErr( {
+                                authId: "ConstellationSecret",
+                                source: "auth",
+                                level: "error",
+                                message: `HTTP error! status: ${response.status} ${response.statusText}`
+                              });
+                            return;
+                          }
```

Comment:
Looks like we need to add the catch block for things like network errors, and then display an error message similar to when the response is not ok?

Reply:
Any un-catched exceptions will be handled bu swagger-ui default functionalities and will be shown on the screen.

Reply:
If I close consty and try to login, it doesn't do anything, no messages displayed. I thought it'd be nice to display a clue to the user. 

Also it lets the user login with an empty string:
<img width="825" height="398" alt="image" src="https://github.com/user-attachments/assets/03553604-aafb-4962-9c21-ac900813e795" />

Reply:
done